### PR TITLE
PT-751 | Change additional criteria field to Activities

### PR DESCRIPTION
**Pull request by quyenlq**
## Additional notes :spiral_notepad:

### CODE REVIEW

File: public/static/locales/en/events.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "labelPlaces": "Places",
     "labelCategories": "Categories",
     "labelTargetGroups": "Target groups",
-    "labelOtherClassification": "Other classification",
+    "labelOtherClassification": "Activities",
```
Comment (nenonja) [inline public/static/locales/en/events.json:30]: `labelActivities` would be more convenient key name?

File: public/static/locales/en/events.json [hunk 2]
```diff
@@ -39,8 +39,8 @@
         "selectedItemRemoveButtonAriaLabel": "Delete category filter"
       },
       "otherClassificationDropdown": {
```
Comment (nenonja) [inline public/static/locales/en/events.json:41]: `activitiesDropdown`

File: public/static/locales/fi/events.json [hunk 3]
```diff
@@ -27,7 +27,7 @@
     "labelPlaces": "Paikat",
     "labelCategories": "Kategoriat",
     "labelTargetGroups": "Kohderyhmät",
-    "labelOtherClassification": "Muu luokittelu",
+    "labelOtherClassification": "Toimintaa",
```
Comment (nenonja) [inline public/static/locales/fi/events.json:30]: perhaps you could ask for translations? I would use `"labelActivities": "Aktiviteetit"`,

File: public/static/locales/fi/events.json [hunk 4]
```diff
@@ -39,8 +39,8 @@
         "selectedItemRemoveButtonAriaLabel": "Poista kategoria"
       },
       "otherClassificationDropdown": {
-        "clearButtonAriaLabel": "Poista muut luokitteluvalinnat",
-        "selectedItemRemoveButtonAriaLabel": "Poista muu luokitteluvalinta"
+        "clearButtonAriaLabel": "Poista toimintaa",
```
Comment (nenonja) [inline public/static/locales/fi/events.json:42]: `Poista aktiviteetit` (plural form) and `Poista aktiviteetti` (single form)

File: public/static/locales/en/events.json (hunk 1 above)
Comment (quyenlq) [reply public/static/locales/en/events.json]: absolutely, fixed :)

**Review by Sartonon — APPROVED** [review]

**Pull request merged**